README.md

```diff
@@ -1,117 +1,167 @@
-
-This should be a fairly short description helps the user decide if your module
-is what they want.
+This module aims to configure a dedicated Puppet Enterprise compiler to become a Bolt Server, with the intention of offloading plan execution from Orchestrator on the Primary server, to Bolt on the Bolt server. A compiler is ideal because it already has access to Puppet Enterprise, code manager, and its local PuppetDB.
 
 ## Setup
 
-### What puppet_bolt_server affects **OPTIONAL**
+### What puppet_bolt_server affects
+
+The `puppet_bolt_server` module will perform the following activities:
 
-If it's obvious what your module touches, you can skip this section. For
-example, folks can probably figure out that your mysql_instance module affects
-their MySQL instances.
+* Install Bolt on the node.
+* Create the `/root/.puppetlabs/etc/bolt/bolt-defaults.yaml` file with custom configuration to:
+    * Use the PCP transport
+    * Use the local PuppetDB for queries
+    * Consume a Puppet token
 
-If there's more that they should know about, though, this is the place to
-mention:
+## Installation
 
-* Files, packages, services, or operations that the module will alter, impact,
-  or execute.
-* Dependencies that your module automatically installs.
-* Warnings or other important notices.
+**Quickstart:** Configure the `puppet_bolt_server` using the PE Console
 
-### Setup Requirements **OPTIONAL**
+This setup will help you to quickly configure the `puppet_bolt_server` in your existing PE server.
 
-If your module requires anything extra before setting up (pluginsync enabled,
-another module, etc.), mention it here.
+1. Add the [puppetlabs-puppet_bolt_server](https://github.com/puppetlabs/puppetlabs-puppet_bolt_server) to your control repo.
+1. Add a new Node Group from the PE Console
 
-If your most recent release breaks compatibility or requires particular steps
-for upgrading, you might want to include an additional "Upgrading" section here.
+```
+  Parent name: PE Infrastructure
+  Group name: Bolt Server
+  Environment: production
+```
 
-### Beginning with puppet_bolt_server
+1. Add the class `puppet_bolt_server` to the Bolt Sever group created in the step above.
+1. Add your compiler dedicated to running Bolt to the group using the Rules tab. Note that this compiler should not be in the compiler pool for catalog compilation.
+1. Add your puppet token (Sensitive string) in the Configuration data tab
+
+    1. We recommend you create a service user inside PE RBAC and choose an appropriate lifetime for its token.
+    1. Generate a token with a lifetime of 1 year: `puppet access login --lifetime 1y`
+
+```
+  Class: puppet_bolt_server
+  Parameter: puppet_token
+  Value: 'insert-your-puppet-token-here'
+```
 
-The very basic steps needed for a user to get the module up and running. This
-can include setup steps, if necessary, or it can be an example of the most basic
-use of the module.
+1. Commit your changes.
```

Comment:
This starts numbering from 1, just like the numbering after the previous code block. . To prevent that, the code blocks should be indented. You can check this in the VS Code Preview tab.

Reply:
Great catch!